### PR TITLE
Warnning about incompatibilities between MIKE+Py and MIKE IO

### PR DESCRIPTION
**Pull request by wuwwen**
Warning about incompatibilities between mikeplus and mikeio.

### CODE REVIEW

File: mikeplus/__init__.py [hunk 1]
```diff
@@ -21,4 +21,8 @@
 
 mike1dio = sys.modules.get("mikeio1d")
 if mike1dio != None:
-    raise RuntimeError('mikeio1d module has been loaded. mikeio1d only can be loaded after mikeplus module.')
\ No newline at end of file
+    raise RuntimeError('mikeio1d module has been loaded. mikeio1d only can be loaded after mikeplus module.')
+
+mikeio = sys.modules.get("mikeid")
```
Comment (ryan-kipawa) [inline mikeplus/__init__.py:26]: typo: mikeio, not mikeid

File: mikeplus/__init__.py [hunk 2]
```diff
@@ -21,4 +21,8 @@
...
+if mikeio != None:
```
Comment (ryan-kipawa) [inline mikeplus/__init__.py:27]: should use 'is not None'

**Review by ryan-kipawa — CHANGES_REQUESTED** [review]
Minor comments.

This will only catch the case where the user imported mikeio before mikepluspy. I think it's reasonable for us to also do this check when they try to do common operations (e.g. load a database).

File: mikeplus/__init__.py (hunk 1 above)
Comment (wuwwen) [reply mikeplus/__init__.py]: Oh, yes. It has been corrected and test successfully by notebook.

**Pull request closed without merging**